user-interface/9001-DATA-data_display.md

```diff
@@ -33,6 +33,15 @@ Vega does not return a `total balance` that is a sum of all accounts in a curren
 
 `Asset.decimals` tells the UIs where to put the decimal place. Ethereum assets often have 18 decimal places, but can have less. Forms where you deposit, withdraw or transfer assets must show all decimal places. It may be appropriate to truncate at a certain number of DP in many cases (e.g. `0.01` instead of `0.012345678912345678` event though `0.001 wBTC` may be worth as much as than `0.01 ETH`). At the moment Vega does not have a source of information that allows conversion of currencies or way of knowing that the significant value of an asset is.
 
+## Quantum
+
+The quantum is a value that is used to define "The minimum economically meaningful amount of this specific asset
+". It can be used to show only the number of decimal places that are meaningful in summary information. A quantum is defined on each asset.
+
+Formula below will make sure that USD is shown to 1 cent .... and for every order of magnitude above / below that we show 1 less / more DP, ie. ETH is big so will show more DP.
+
+`Number DP = MAX(0, LOG10(100/Quantum))`
```

Comment:
This makes sense to me however there is an edge case here.

1x10^-18 vega is not a meaningful amount of money, so if I had this associated for some reason this formula would round the amount down to zero even if the user had a non-zero balance. Are we ok with this trade off?

Reply:
Perhaps a more meaninful example is, rewards are paid out based on share. Many people will have very small shares, 0.000002% or something. On the rewards page it is likely if my share is not big I will appear to have earnt no rewards with this formula if the amount distributed to me is below quantum.

Reply:
The more generic point I guess I am trying to make is that the amount of DP to show can be dependent on multiple things:

* The amount the user has, if I am a millionaire I don't care about pence. If I'm not so wealthy I really do, possibly even less than pence.
* The amount the asset is worth, quantum
* The amount of decimal places a UI can support. It would be great to show 18 d.p. for everything but if there is a max supply of billions 27 digits is probably a bit much

Only considering one of these means that we may end up with overflow errors and misrepresenting small user balances. That's fine, but as long as we have said we are ok with that.